src/pages/index.js

```diff
@@ -212,54 +42,366 @@ const IndexPage = () => (
+            >
+              <SprkStackItem>
+                <SprkHeading element="h3" variant="displayFive">
+                  Components
+                </SprkHeading>
+              </SprkStackItem>
+
+              <SprkStackItem>
+                <SprkText variant="bodyTwo">
+                  Explore the library of Spark components and supporting
+                  documentation, including design guidelines, code samples, and
+                  configuration options.
+                </SprkText>
+              </SprkStackItem>
+
+              <SprkStack
+                additionalClasses="
+                  sprk-o-Stack__item
+                  sprk-o-Stack--end-column"
+              >
+                <SprkStackItem>
+                  <SprkLink
+                    variant="unstyled"
+                    additionalClasses="sprk-c-Button sprk-c-Button--secondary"
+                    href="/using-spark/components/button"
+                  >
+                    Go To Components
+                  </SprkLink>
+                </SprkStackItem>
+              </SprkStack>
+            </SprkStack>
+          </SprkCard>
+
+          <SprkCard additionalClasses="docs-c-Card">
+            <SprkStackItem>
+              <SprkLink
+                variant="unstyled"
+                href="/using-spark/guides"
+                analyticsString="teaser-media"
+              >
+                <img
+                  className="sprk-c-Card__media"
+                  alt="Explore Guides"
+                  src={guidesMedia}
+                />
+              </SprkLink>
+            </SprkStackItem>
+
+            <SprkStack
+              additionalClasses="sprk-o-Stack__item sprk-c-Card__content"
+              itemSpacing="large"
+            >
+              <SprkStackItem>
+                <SprkHeading element="h3" variant="displayFive">
+                  &apos;How To&apos; Guides
+                </SprkHeading>
+              </SprkStackItem>
+
+              <SprkStackItem>
+                <SprkText variant="bodyTwo">
+                  Guidance on how to make design choices that prioritize user
+                  experience. Get answers on topics like layout, messaging, and
+                  placement.
+                </SprkText>
+              </SprkStackItem>
+
+              <SprkStack
+                additionalClasses="
+                  sprk-o-Stack__item
+                  sprk-o-Stack--end-column"
+              >
+                <SprkStackItem>
+                  <SprkLink
+                    variant="unstyled"
+                    additionalClasses="sprk-c-Button sprk-c-Button--secondary"
+                    href="/using-spark/guides"
+                  >
+                    Explore Guides
+                  </SprkLink>
+                </SprkStackItem>
+              </SprkStack>
+            </SprkStack>
+          </SprkCard>
+
+          <SprkCard additionalClasses="docs-c-Card">
```

Comment:
Whats this `docs-c-Card` class do? I forgot, wondering if we need it anymore?

Reply:
This adds some flexbox stuff that's specific to the docs site cards and it's children.

Reply:
okay, thanks